frontend/src/metabase/embedding/embedding-iframe-sdk/analytics.ts

```diff
@@ -86,12 +103,16 @@ export function createEmbeddedAnalyticsJsUsage(
             drills: { true: 0, false: 0 },
             with_downloads: { true: 0, false: 0 },
             with_title: { true: 0, false: 0 },
-          });
+            with_subscriptions: { true: 0, false: 0 },
+          } satisfies EmbeddedAnalyticsJsEvent["dashboard"]);
```

Comment:
The `satisfies` part ensures TypeScript would throw an error when we change the upstream type, but forgot to update this default values.